apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue

```diff
@@ -378,10 +378,14 @@ const handleClickRow = (rowData) => {
         white-space: nowrap;
 
         .th-contents {
-            @apply flex justify-between pl-4;
+            @apply flex justify-between;
+            padding: 0.25rem 0 0.25rem 1rem;
             line-height: 2;
             .th-text {
                 @apply inline-flex items-center gap-1;
+                white-space: pre-line;
+                text-align: right;
+                line-height: 1.25;
```

Comment:
description: for this

![스크린샷 2023-06-13 오후 8 07 42](https://github.com/cloudforet-io/console/assets/18563857/99d3d4db-4c67-42e1-93fc-24192ebeab81)